pkgs/development/python-modules/pyopen-wakeword/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  autoPatchelfHook,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  numpy,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pyopen-wakeword";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rhasspy";
+    repo = "pyopen-wakeword";
+    tag = "v${version}";
+    hash = "sha256-LJ0pHP4nsTx3GPuWUwOwNuXR9tUKABqSHnLSvMlfm1Y=";
+  };
+
+  postPatch = ''
+    python ./script/copy_lib
```

```suggestion
    ${python.interpreter} ./script/copy_lib
```
or does that cause problems when cross-compiling?